sources/packages/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -591,4 +597,68 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
       })
       .getOne();
   }
+
+  /**
+   * Get all offerings that were were requested for change
```

Comment:
Type on "were were".

Reply:
This one does not seem changed to me. Please search for "were were", there are two currencies to be fixed.